### PR TITLE
Fix errors in CLI config file docs

### PR DESCRIPTION
**Pull request by msullivan**

### CODE REVIEW

**Review by elprans — APPROVED** [review]

File: docs/cli/index.rst [hunk 1]
```diff
@@ -118,13 +118,14 @@ The ``cli.toml`` has the following structure. All fields are optional:
...
     implicit-properties = false   # Print implicit properties of objects
-    implicit-limit = 100          # Set implicit LIMIT
+    limit = 100                   # Set implicit LIMIT
                                   # Defaults to 100, specify 0 to disable
     input-mode = "emacs"          # Set input mode. One of: vi, emacs
     output-format = "default"     # Set output format.
                                   # One of: default, json, json-pretty,
                                   # json-lines
-    print-stats = false           # Print statistics on each query
+    print-stats = "off"           # Print statistics on each query.
```
Comment (1st1) [inline docs/cli/index.rst:127]: @tailhook @fantix 

This seems like a bug -- why `"off"` here and `false` for other bool props?

Comment (colinhacks) [reply docs/cli/index.rst:127]: This is a ternary option "off"/"query"/"detailed" 

https://github.com/edgedb/edgedb-cli/blob/3aa3b0cb59e6455aaf0a856d9615ee12a81782d5/src/repl.rs#L38

Comment (1st1) [reply docs/cli/index.rst:127]: Perhaps we should standardize other bool prefs to "on" / "off". Or the users will be confused.

Comment (1st1) [reply docs/cli/index.rst:127]: Perhaps we should standardize other bool prefs to "on" / "off". Or the users will be confused.

@msullivan The setting help should list all valid setting values.

Comment (msullivan) [reply docs/cli/index.rst:127]: Yeah that's the next line.

Comment (tailhook) [reply docs/cli/index.rst:127]: @1st1 
> Perhaps we should standardize other bool prefs to "on" / "off". Or the users will be confused. 

We do allow `on`/`off` in the shell. But this is toml. Boolean values should be boolean values as specified by toml (so you don't have to use quotes and generally it's friendly for people knowing toml).

**Pull request merged**